app/build.gradle

```diff
@@ -13,13 +13,21 @@ android {
             jsonFile = file('app-publisher.json')
         }
     }
+    signingConfigs {
+        release {
+            storeFile file('../SunsetScrob.jks')
+            storePassword System.getenv('SUNSET_STORE_PASSWORD')
+            keyAlias "sunsetScrob"
+            keyPassword System.getenv('SUNSET_KEY_PASSWORD')
+        }
+    }
     defaultConfig {
         applicationId "com.mataku.scrobscrob"
 
         minSdkVersion 24
         targetSdkVersion 26
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/d98de3fb9ea0ace0e5c0241379c958a7804774f9/app/build.gradle#L28"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>